### PR TITLE
Add GA4 banner docs and update page view event

### PR DESCRIPTION
**Pull request by AshGDS**
Adds the GA4 banner docs, and update page view event

### CODE REVIEW

**Review by andysellick — COMMENTED** [review]
Always happy when someone updates the implementation record 🎉 

Just a few questions/suggestions.

File: data/analytics/events.yml [hunk 1]
```diff
@@ -78,19 +78,60 @@
...
 - name: banners
+  description: "Note that the presence of banners is tracked via our Pageview tracker."
   events:
-    - name: banner shown
+    - name: link click
+      description: Clicking a link within a banner
+      examples:
+        - text: The "BETA" banner on this page contains the link "find out what beta means"
+          link: https://www.gov.uk/hmrc-internal-manuals/debt-management-and-banking
       implemented: false
```
[the remark this answers is not in the document]
Comment (AshGDS) [reply data/analytics/events.yml]: Good spot, thanks!

File: data/analytics/events.yml [hunk 2]
```diff
@@ -78,19 +78,60 @@
...
       priority: medium
       data:
       - name: event
         value: event_data
+      - name: event_data
+        value:
+        - name: event_name
+          value: select_content
+        - name: type
+          value: banner type e.g. intervention/phase/cookie banner
```
[the remark this answers is not in the document]
Comment (AshGDS) [reply data/analytics/events.yml]: Good spot, thanks

File: data/analytics/events.yml [hunk 3]
```diff
@@ -78,19 +78,60 @@
...
+          - name: index_link
+        - name: index_total
+        - name: section
+          value: title of the banner
+        - name: external
+        - name: method
+        - name: link_domain
+        - name: link_path_parts
+    - name: dismiss button
+      description: Clicking a dismiss button to hide the banner
```
[the remark this answers is not in the document]
Comment (AshGDS) [reply data/analytics/events.yml]: Thanks, updated :+1:

**Review by andysellick — APPROVED** [review]

**Pull request merged**